src/test/skript/junit/ExprHoverList.sk

```diff
@@ -0,0 +1,12 @@
+test "ExprHoverListJUnit" when running JUnit:
+	set {_tests::1} to "clear list"
+
+	ensure junit test "org.skriptlang.skript.test.tests.syntaxes.ExprHoverListTest" completes {_tests::*}
+
+on server list ping:
+	set {_test} to "org.skriptlang.skript.test.tests.syntaxes.ExprHoverListTest"
+	junit test is {_test}
+
+	clear hover list
+	if size of hover list is 0:
+		complete objective "clear list" for {_test}
```

Comment:
```suggestion
		complete objective "clear list" for {_test}

```

Reply:
@sovdeeth check [this comment](https://github.com/SkriptLang/Skript/pull/6988#issuecomment-23048115630) for why it doesn't work